### DISCUSSION TITLE
Overflow: Maximum Recursion Level Reached

### DISCUSSION THREAD

**Discussion by BenLyddane** [PS1607/mbox-to-json#8] [General]
After initializing and getting to 100% I am getting this error: 

OverflowError: Maximum recursion level reached

I tried chaning my recursion limit to 10,000 , but still got this error. Wondering if there's a problem with large files? (Mine is ~6.5 gb)

**Comment by PS1607** [comment]
Hi @BenLyddane 

I didn't have enough resources to test it for a bigger file. You can try by chunking down the MBOX file in multiple small parts and then try? I will try to look into this issue when I get time.
